src/jvm/main/org/jetbrains/kotlinx/lincheck/strategy/managed/modelchecking/ModelCheckingStrategy.kt

```diff
@@ -229,30 +229,30 @@ internal class ModelCheckingStrategy(
                     }
 
                     if (representation.isNotEmpty()) {
-                        representations.add("$type;${node.iThread};${node.callDepth};${node.shouldBeExpanded(false)};${eventId};${representation}")
+                        representations.add("$type;${node.iThread};${node.callDepth};${node.shouldBeExpanded(false)};${eventId};${representation};null")
```

Comment:
Why do you need this `null`?

Reply:
It is a stack trace element according to plugin, I don't know if it is used aywhere

Reply:
I'm afraid, this change might break the Lincheck plugin and is unnecessary for the Trace Debugger one.

Reply:
@avpotapov00 do you know how this argument is used in the trace debugger plugin?